app/models/validator_score_v1.rb

```diff
@@ -64,6 +64,13 @@ class ValidatorScoreV1 < ApplicationRecord
   serialize :skipped_slot_history, JSON
   serialize :skipped_after_history, JSON
 
+  def skipped_slot_history_moving_averages
+    ValidatorBlockHistory
+      .where(validator: validator, network: network)
+      .last(60)
```

Comment:
[The front-end only looks at the last 60 records](https://github.com/brianlong/www.validators.app/blob/master/app/views/validators/index.html.erb#L242-L247) so we only need to load 60.